test/e2e/openshift/openshift_test.go

```diff
@@ -85,6 +85,62 @@ var _ = Describe("Azure Container Cluster using the OpenShift Orchestrator", fun
 		}
 	})
 
+	It("should whitelist pods on compute nodes", func() {
+		namespaces := []string{
+			"openshift-infra",
+			"openshift-node",
+			"openshift-metrics",
+			"kube-",
+			"default",
+		}
+
+		whitelist := []string{
```

Comment:
daemonsetPrefixes

Reply:
This could also include `statefulsets`.  Do you still prefer `daemonsetPrefixes`?